tests/phpunit/api/v3/PaymentTest.php

```diff
@@ -104,13 +106,15 @@ public function testCreatePaymentNoLineItems() {
       'contribution_id' => $contribution['id'],
       'total_amount' => 50,
     );
-    $payment = $this->callAPIAndDocument('payment', 'create', $params, __FUNCTION__, __FILE__);
+    $payment = $this->callAPISuccess('payment', 'create', $params);
```

Comment:
I know I said to use callAPIAndDocument less - but I hope you are generating one example for each action.